### PR TITLE
LRDOCS-7369 Content Pages vs Widget Pages

### PR DESCRIPTION
**Pull request by codyhoag**
Hey Jorge; can you review this content, which is based on the info you provided in this [Google Doc](https://docs.google.com/document/d/1Te7fNVxZHBIVgyhYQIlYiXZsb52scaERidkShcV1FJc/edit)?

I wanted to give the context I was providing this info for in our docs, so I figured a PR would be easier. Please let me know any feedback you have. Thanks!

### CODE REVIEW

File: en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/01-adding-pages-to-sites-intro.markdown [hunk 1]
```diff
@@ -12,3 +12,52 @@ fact, sites primarily exist for the sake of organizing pages and content, so
...
+
+[*Content Pages*](/docs/7-2/user/-/knowledge_base/u/creating-content-pages): the
+new page type providing more flexibility, especially for non-technical users.
+
+[*Widget Pages*](/docs/7-2/user/-/knowledge_base/u/adding-applications-to-a-page):
+the traditional page type that is more rigid. There may still be some use cases
+where it's preferred.
+
+You should always opt for Content Pages, unless there's a concrete reason
+otherwise. Content Pages are being actively developed and offer many key
```
Comment (JorgeFerrer) [inline en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/01-adding-pages-to-sites-intro.markdown]: Reading "are being actively developed" makes me think that other features are not, and potentially also that they are still half baked. I wonder if there is some other way of wording this that just praises Content Pages but doesn't leave this taste.

File: en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/01-adding-pages-to-sites-intro.markdown [hunk 2]
```diff
@@ -12,3 +12,52 @@ fact, sites primarily exist for the sake of organizing pages and content, so
...
+and
+[A/B Testing](/docs/7-2/user/-/knowledge_base/u/a-b-testing). Furthermore,
+Content Pages are easier to use and are more powerful for non-technical users
+compared to Widget Pages.
+
+Why would someone prefer Widget Pages? Usually Widget Pages are preferred
+because they were the only page type available in earlier versions of @product@,
+so they're more familiar than Content Pages.
+
+In addition, there are several other things that Widget Pages provide that are
```
Comment (JorgeFerrer) [inline en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/01-adding-pages-to-sites-intro.markdown]: "there are still a few things that Widget Pages ..."

File: en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/01-adding-pages-to-sites-intro.markdown [hunk 3]
```diff
@@ -12,3 +12,52 @@ fact, sites primarily exist for the sake of organizing pages and content, so
...
+not possible with Content Pages:
+
+- *Developing an advanced custom layout*: Content Pages let page authors create
+  their own page layout. As a result, developers can't create their own custom
+  layouts with FreeMarker like Widget Pages allowed for.
```
Comment (JorgeFerrer) [inline en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/01-adding-pages-to-sites-intro.markdown]: Add "Content Pages instead provides the ability to create a layout visually, which we believe is a better approach, but the programmatic approach allows for more advanced capabilities"

File: en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/01-adding-pages-to-sites-intro.markdown [hunk 4]
```diff
@@ -12,3 +12,52 @@ fact, sites primarily exist for the sake of organizing pages and content, so
...
+- *User-Customizable columns*: This was a rarely used feature of Widget Pages
+  that is not provided in Content Pages. If your page requires a
+  user-customizable column, you must leverage a Widget Page.
+
+- *Using Staging with Page Variations*: Content Pages do not support Staging's
+  Page Variations. This avoids possible confusion with the similar capability to
+  create variations of a page that are used in personalization and A/B Testing.
+
+- *Widget permissions*: You can configure widget permissions on a Widget Page.
+  This is not possible for Content Pages.
```
Comment (JorgeFerrer) [inline en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/01-adding-pages-to-sites-intro.markdown]: "This capability is not yet possible for Content Page, although it's planned for a future release"

**Comment by JorgeFerrer** [comment]
I've reviewed the content and it's pretty good. Once it's published, could you create a Loop post and a blog post with a summary of it pointing to the documentation?

**Comment by codyhoag** [comment]
Thanks, Jorge! I've incorporated your feedback. I'll send for publication today!

**Pull request closed without merging**